bounded-collections/src/lib.rs

```diff
@@ -41,15 +41,33 @@ pub trait Get<T> {
 	fn get() -> T;
 }
 
-impl<T: Default> Get<T> for () {
+/// A trait for querying a single value from a type.
+///
+/// The value should be a constant.
+pub trait ConstGet<T> {
+	/// Return the constant value.
+	fn get() -> T;
+}
+
+/// Implement Get for any type that implements ConstGet.
+impl<T, U> Get<U> for T
```

Comment:
This implementation can be omitted if we just make `Get<T>` a supertrait of `ConstGet<T>`.

Reply:
Done, thanks!